cache/blobs.go

```diff
@@ -521,6 +519,7 @@ func commitOverlaybd(ctx context.Context, sr *immutableRef, desc *ocispecs.Descr
 	}
 	desc.Digest = dgst
 	desc.Size = sz
+	desc.MediaType = ocispecs.MediaTypeImageLayer
```

Comment:
I moved this here, as `commitOverlayBD` was already mutating the descriptor 🤔

Reply:
Yes, I think it is more appropriate to put together the modification of overlaybd for desc.